### PR TITLE
Fix: remove the dependence of EXX on RPA

### PR DESCRIPTION
**Pull request by maki49**
Whether to output the bare Coulomb matrix (for RPA) should not influence EXX parameters in SCF calculation. 
Before this fix, `dft_functional==HSE/PBE0 && rpa==1` was problematic due to the mismatch between `dft_functional` and `ccp_type`.

### CODE REVIEW

**Review by PeizeLin — APPROVED** [review]

File: docs/advanced/input_files/input-main.md [hunk 1]
```diff
@@ -2283,6 +2284,12 @@ These variables are relevant when using hybrid functionals.
   - True: if gamma_only 
   - False: else 
 
+### rpa_ccp_rmesh_times
```
Comment (mohanchen) [inline docs/advanced/input_files/input-main.md:2280]: the variable name is too long

File: source/module_io/write_input.cpp [hunk 2]
```diff
@@ -398,6 +398,7 @@ ModuleBase::GlobalFunc::OUTP(ofs, "out_bandgap", out_bandgap, "if true, print ou
     ModuleBase::GlobalFunc::OUTP(ofs, "exx_opt_orb_lmax", exx_opt_orb_lmax, "the maximum l of the spherical Bessel functions for opt ABFs");
     ModuleBase::GlobalFunc::OUTP(ofs, "exx_opt_orb_ecut", exx_opt_orb_ecut, "the cut-off of plane wave expansion for opt ABFs");
     ModuleBase::GlobalFunc::OUTP(ofs, "exx_opt_orb_tolerence", exx_opt_orb_tolerence, "the threshold when solving for the zeros of spherical Bessel functions for opt ABFs");
+    ModuleBase::GlobalFunc::OUTP(ofs, "rpa_ccp_rmesh_times", rpa_ccp_rmesh_times, "how many times larger the radial mesh required for calculating Columb potential is to that of atomic orbitals");
```
Comment (mohanchen) [inline source/module_io/write_input.cpp:402]: the English expression should be improved, it should not start with "how many times" in this sentence

File: source/module_ri/RPA_LRI.hpp [hunk 3]
```diff
@@ -71,6 +71,11 @@ void RPA_LRI<T, Tdata>::cal_postSCF_exx(const elecstate::DensityMatrix<T, Tdata>
 			? RI_2D_Comm::split_m2D_ktoR<Tdata>(kv, mix_DMk_2D.get_DMk_gamma_out(), *dm.get_paraV_pointer())
 			: RI_2D_Comm::split_m2D_ktoR<Tdata>(kv, mix_DMk_2D.get_DMk_k_out(), *dm.get_paraV_pointer());
 
+    // set parameters for bare Coulomb potential
+    GlobalC::exx_info.info_global.ccp_type = Conv_Coulomb_Pot_K::Ccp_Type::Hf;
```
Comment (mohanchen) [inline source/module_ri/RPA_LRI.hpp:75]: Is it possible to remove GlobalC::exx_info? We should not use GlobalC in future.

File: docs/advanced/input_files/input-main.md (hunk 1 above)
Comment (maki49) [reply docs/advanced/input_files/input-main.md:2280]: It is to be consistent with `exx_ccp_rmesh_times`.

File: source/module_io/write_input.cpp (hunk 2 above)
Comment (maki49) [reply source/module_io/write_input.cpp:402]: I think it's a phrase rather than a sentence.

File: source/module_ri/RPA_LRI.hpp (hunk 3 above)
Comment (maki49) [reply source/module_ri/RPA_LRI.hpp:75]: OK, I will do it in another PR.

**Review by mohanchen — APPROVED** [review]

**Pull request merged**